.changeset/wet-eyes-buy.md

```diff
@@ -0,0 +1,7 @@
+---
+'@evidence-dev/evidence': patch
+'@evidence-dev/preprocess': patch
+'@evidence-dev/components': patch
+---
```

Comment:
@Winterhart changesets automatically added all of these packages, though I'm not sure all are needed.

I would have suspected just components?

Should I remove the others do you think?